### DISCUSSION TITLE
Camera

### DISCUSSION THREAD

**Discussion by StonedHackerman** [dotnet/maui#5532] [Q&A]
I know XTC has a CameraView, but is there an official plan to port it or write a CameraView, and/or some other API for controlling the camera, directly for MAUI? I know it might be complicated (Android, looking at you), but this is definitelly something that the Essentials should contain.

And if it is planned, will it be in the release candidate version, or only in GA?

Thank you very much

**Comment by pictos** [comment]
@StonedHackerman AFAIK MAUI will not have a CameraView control, we will port the XCT implementation over the MCT. But that will be available after GA, because we don't have resources to implement that before.
